net2/HostManager.js

```diff
@@ -950,6 +951,19 @@ module.exports = class HostManager {
     json.networkProfiles = await NetworkProfileManager.toJson();
   }
 
+  async networkMetrics(json) {
+    const nmkeys = await rclient.keysAsync("metric:throughput:stat:*");
```

Comment:
keysAsync will be expensive and it will block other redis calls.

Reply:
Fix this and all found occurrance of same logic(get physical ethernet interfaces) with following line

```
ls -l /sys/class/net | awk '/^l/ && !/virtual/ {print $9}'
```